bindgen-integration/cpp/Test.h

```diff
@@ -178,3 +178,7 @@ struct AutoRestoreBool {
 struct WithWChar {
   wchar_t foo[30];
 };
+
+const int MY_PREFIXED_CONST_VALUE = 3;
```

Comment:
It may be worth to point out in a comment that these are unprefixed by the parse callbacks.